float64[] max_effort
---
int8 err
